package.json

```diff
@@ -29,7 +29,7 @@
     "mocha-eslint": "^4.1.0",
     "mysql2": "^1.4.2",
     "nodemailer": "^4.4.0",
-    "sequelize": "^4.22.5",
+    "sequelize": "^4.37.6",
```

Comment:
Why are you updating the version of Sequelize in a PR about a welcome controller. This should be a separate PR with requests for specific review of database interactions. Additionally, did you fully vet the impact of this update across the entire system's use of the database?

Reply:
Mistake while re-installing packages, should be amended in latest commit.